src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -147,6 +148,19 @@ class RequestorStep extends React.Component {
         BankAccounts.updatePersonalInformationForBankAccount(payload);
     }
 
+    submitOnfidoVerification() {
+        if (!this.validate()) {
+            return;
+        }
+
+        BankAccounts.verifyIdentityForBankAccount(
+            ReimbursementAccountUtils.getDefaultStateForField(this.props, 'bankAccountID', 0),
+            {
+                ...this.state,
```

Comment:
verifyIdentityForBankAccount doesn't need to pass the whole state. Just onfidoData